src/resolvers/user_management.js

```diff
@@ -30,22 +30,75 @@ export default {
       //   throw new Error('Already logged in.')
       // }
       const session = driver.session()
-      return session.run(
-        'MATCH (user:User {email: $userEmail}) ' +
-        'RETURN user {.id, .slug, .name, .avatar, .email, .password, .role} as user LIMIT 1', {
-          userEmail: email
-        })
-        .then(async (result) => {
+      return session
+        .run(
+          'MATCH (user:User {email: $userEmail}) ' +
+            'RETURN user {.id, .slug, .name, .avatar, .email, .password, .role} as user LIMIT 1',
+          {
+            userEmail: email
+          }
+        )
+        .then(async result => {
           session.close()
           const [currentUser] = await result.records.map(function (record) {
             return record.get('user')
           })
 
-          if (currentUser && await bcrypt.compareSync(password, currentUser.password)) {
+          if (
+            currentUser &&
+            (await bcrypt.compareSync(password, currentUser.password))
+          ) {
             delete currentUser.password
             return encode(currentUser)
-          } else throw new AuthenticationError('Incorrect email address or password.')
+          } else {
+            throw new AuthenticationError(
+              'Incorrect email address or password.'
+            )
+          }
         })
+    },
+    changePassword: async (
+      _,
+      { oldPassword, newPassword },
+      { driver, user }
+    ) => {
+      const session = driver.session()
+      let result = await session.run(
+        `MATCH (user:User {email: $userEmail}) 
+         RETURN user {.id, .email, .password}`,
+        {
+          userEmail: user.email
+        }
+      )
+
+      const [currentUser] = result.records.map(function (record) {
+        return record.get('user')
+      })
+
+      if (!(await bcrypt.compareSync(oldPassword, currentUser.password))) {
+        throw new AuthenticationError('Old password isn\'t valid')
```

Comment:
```suggestion
        throw new AuthenticationError('Old password is not correct')
```
For written English, I try to avoid abbreviations

Reply:
Gotcha